createdisk.sh

```diff
@@ -113,7 +113,7 @@ EOF
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/100-crio-bridge.conf'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/200-loopback.conf'
 
-podman_version=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'podman version -f json | jq -r .Client.Version')
+podman_version=${PODMAN_VERSION:-$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'podman version -f json | jq -r .Client.Version')}
```

Comment:
this would mean it is always an override for the one inside the VM ...

... but I am at that moment also concerned that the version from the VM returns `-dev` ?

Reply:
Yes this need to be removed from CI script as soon as we cut a branch so that CI on that branch will always take what part of VM and nightly always takes what provided in CI script.